Assets/ObstacleTower/Scenes/Procedural.unity

```diff
@@ -689,7 +694,7 @@ MonoBehaviour:
   m_Model: {fileID: 0}
   m_InferenceDevice: 0
   m_BehaviorType: 0
-  m_BehaviorName: ObstacleTowerAgent
+  m_BehaviorName: ObstacleTowerAgent-v4.0
```

Comment:
Is there a reason for adding the version to the behavior name ?

Reply:
We use the version to check for compatibility. In the past this was handled through the academy name, but that was deprecated, so now we check for it here.